setup.py

```diff
@@ -35,5 +35,10 @@ def load_reqs(filename):
     include_package_data=True,
     packages=find_packages(),
     install_requires=requirements,
-    tests_require=test_requirements
+    tests_require=test_requirements,
+    entry_points={
+        'console_scripts': [
+            'initdb=guillotina_cms.scripts:initdb',
```

Comment:
fwiw, we have guillotina commands where we can easily do these things!

Reply:
yeah, @bloodbare told me but he said that might be overkill and it would take me a while as I would have to deal with other inner Guillotina things, so I went the easy way...